interview-questions.md

```diff
@@ -8,48 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Researched answer:
+Researched answer: JSX is a markdown syntax extension to Javascript and it is combining the logic and the markup together. One syntax difference between the two are the empty fragments and also, you must return every parent element. Tags can also self close such as a component call, unlike HTML everything needs an opening and closing tag. 
+
+<!-- Sources from https://www.freecodecamp.org/news/html-vs-jsx-whats-the-difference/#:~:text=HTML%20is%20a%20very%20important,a%20syntactic%20sugar%20for%20React. -->
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: Yarn stands for Yet Another Resource Negotiator and is the file manager for Javascript. I believe node modules are modified. 
 
-Researched answer:
+Researched answer: I'm not exactly sure what the question means, but when yarn is added there a bunch of files that are added such as the node modules which are all the dependencies that are needed for React App. 
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: An anonymous function is an empty function that we need when we don't have a return statement for it.
 
-Researched answer:
+Researched answer: Anyonymous functions are functions without a name. It seems these are used when we don't have a need for the function at the moment so instead we use a log to show the variable.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: State values are where the values for the data are stored and props are how we access that data. 
+
+Researched answer: I've found that state values cannot be accessed or modified outside of the componenet and that props make componenets reuseable by giving the ability to get data from the parent componenent in the form of props. 
 
-Researched answer:
+(I think I was close?)
 
 5. What is the DOM?
 
-Your answer:
+Your answer: Document Object Model and it is any visual representation of code such as a website. 
 
-Researched answer:
+Researched answer: What this means is any website that shows anything visual is a DOM, unless it's a blank screen than it would not be a DOM.
 
 6. STRETCH: Which is the difference between a div and a span?
 
-Your answer:
+Your answer: I know div is used for a division of space and at this I'm not 100% sure what a span is, but I would like to say I believe it is similar to a div but in a bigger purpose meaning it holds more possibly. 
 
-Researched answer:
+Researched answer: div is used for the styling of elements on a page and also is a block-level organization which means it's a block of text that can be inputted into CSS (for example) and that html element or react element can be used to style that section. Span is used in actually a smaller way meaning you can only style a certain section that you want such as <p> content, content <span> content </span> </p>. 
```

Comment:
Good job, Zack!  I really liked your answer to #6!  Excellent job of using examples throughout your questions.  Keeping working on that vocab - I definitely see improvement!